src/app/health/health-update.component.ts

```diff
@@ -31,7 +32,7 @@ export class HealthUpdateComponent implements OnInit {
     this.initProfileForm();
     this.initHealthForm();
     const currentYear = new Date().getFullYear();
-    this.minDate = new Date(currentYear - 120, 0, 1);
+    this.minBirthDate = new Date(currentYear - 120, 0, 1);
```

Comment:
@AbdulkadirAbdullahi This should not be relative to the current year.  The minimum date should be January 1, 1900.